server/middlewares/passport/passportStrategies.js

```diff
@@ -0,0 +1,23 @@
+import socialAuthConfig from '<configs>/passport';
```

Comment:
I already know this is the passport folder why do you have to name the file `passportStrategies`, strategies is enough